### PR TITLE
PyData London 2024

### PR DESCRIPTION
**Pull request by SaranjeetKaur**
Note: the avatars used in the yml are from GitHub profiles - this might need a change to Imperial profiles photos.

The `.authors.yml` is populated alphabetically with info from the [team's web page](https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/service-offering/research-software-engineering/about-the-team/).

### CODE REVIEW

**Comment by SaranjeetKaur** [comment]
@Sahil590 - looks like all the relevant content is on this PR now. I took the liberty to request for reviews. Please feel free to make any changes that you feel are appropriate.

File: docs/.authors.yml [hunk 1]
```diff
@@ -1,9 +1,45 @@
...
+    name: Sahil Raja
+    description: Junior Software Developer
+    avatar: https://github.com/Sahil590.png
+  SaranjeetKaur:
+    name: Saranjeet Kaur Bhogal
+    description: Research Software Engineer
+    avatar: https://github.com/SaranjeetKaur.png
+  tsmbland:
+    name: Tom Bland
+    description: Research Software Engineer - HPC and RSE Experience Programme
```
Comment (dalonsoa) [inline docs/.authors.yml:44]: We are about to change his job title.
```suggestion
    description: Research Software Engineer
```

File: docs/.authors.yml
Comment (dalonsoa) [inline docs/.authors.yml:1]: Yes, use the group webpage photos, for consistency.

File: docs/posts/pydata_london_2024.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  - AdrianDAlessandro
+  - HarmonicReflux
+  - jamesturner246
+  - SaranjeetKaur
+  - tsmbland
+categories:
+  - Events
+tags:
+  - Conference
+  - Python
```
Comment (dalonsoa) [inline docs/posts/pydata_london_2024.md:13]: The post is already in the `Events` category, so I'm not sure if it is useful to add a `Conference` tag as well. Maybe replace with `PyData` directly. Also, maybe add `Open source` and `Machine learning` as it describes the content of several of the talks you mention.

**Review by dalonsoa — CHANGES_REQUESTED** [review]
Looks good to me! I've just suggested a couple minor changes.

File: docs/posts/pydata_london_2024.md (hunk 2 above)
Comment (SaranjeetKaur) [reply docs/posts/pydata_london_2024.md:13]: Done!

File: docs/.authors.yml
Comment (SaranjeetKaur) [reply docs/.authors.yml:1]: done!

**Review by dalonsoa — APPROVED** [review]

**Comment by AdrianDAlessandro** [comment]
This looks fantastic! And obviously is already merged, but one thing I'm curious about is if the images can be formatted to be centred and or made a specific size? They are currently all left-aligned and are different sizes.

It would be ideal if we could make it a global setting (maybe in a stylesheet) to align images in a specific way.

**Comment by SaranjeetKaur** [comment]
> This looks fantastic! And obviously is already merged, but one thing I'm curious about is if the images can be formatted to be centred and or made a specific size? They are currently all left-aligned and are different sizes.
> 
> It would be ideal if we could make it a global setting (maybe in a stylesheet) to align images in a specific way.

Thanks @AdrianDAlessandro! We were wondering if we should re-size and align the images for consistency, because as of now we have used the images as they are (which it is not a good approach in the long run). I will create a separate issue based on your comment.

**Pull request merged**